src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,52 @@
+package duke;
+
+import duke.command.Command;
+
+/**
+ * Represents a Personal Assistant Chatbot that helps a person to keep track of various tasks.
+ */
+public class Duke {
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Creates a new Duke Chatbot that saves and loads tasks from the given filepath.
+     * @param filePath The file path to load tasks from and save tasks to.
+     */
```

Comment:
You might have missed out the blank comment line between the description and the list of tags